src/lib/client.ts

```diff
@@ -734,6 +731,7 @@ export default class MqttClient extends TypedEventEmitter<MqttClientEventCallbac
 	 * @param Function callback call when finished
 	 * @api public
 	 */
+	// @ts-expect-error - packet is unused. @TODO: This method is seemingly never called.
```

Comment:
This is never used but can be overridden by user in order to handle messages with backpressure on stream (instead of using `message` event that doesn't provide backpressure)